data/forms/add-relationship.tsx

```diff
@@ -16,42 +16,42 @@ const formSteps: FormStep[] = [
         component: 'Select',
         placeHolder: 'Relationship type',
         name: 'type',
-        label: 'Type',
+        label: 'Relationship type',
+        labelSize: 's',
         width: 20,
         rules: { required: true },
-        options: RELATIONSHIPS,
+        options: RELATIONSHIPS.sort((a, b) => a.text.localeCompare(b.text)),
       },
       {
-        component: 'Radios',
-        name: 'ofUnbornChild',
-        label: 'Of an unborn child',
         conditionalRender: ({ type }) =>
-          type === 'parent' || type === 'sibling',
-        rules: { required: true },
-        options: [
-          { value: 'Y', text: 'Yes' },
-          { value: 'N', text: 'No' },
-        ],
-      },
-      {
-        component: 'Radios',
-        name: 'isMainCarer',
-        label: 'Main carer',
-        rules: { required: true },
-        options: [
-          { value: 'Y', text: 'Yes' },
-          { value: 'N', text: 'No' },
-        ],
-      },
-      {
-        component: 'Radios',
-        name: 'isInformalCarer',
-        label: 'Informal carer',
-        rules: { required: true },
-        options: [
-          { value: 'Y', text: 'Yes' },
-          { value: 'N', text: 'No' },
-        ],
+          type !== 'unbornSibling' &&
+          type !== 'unbornChild' &&
+          type !== '' &&
+          type !== undefined,
+        component: 'Checkbox',
+        name: 'additionalOptions',
+        label: 'Select all that apply',
+        labelSize: 's',
+        options: ({ type }) => {
+          const typeOptions: Option[] = [
+            { value: 'isMainCarer', text: 'Is a main carer' },
+          ];
+
+          if (type === 'parent') {
```

Comment:
These conditionals are cool but I wonder if it could be confusing.

Will a worker know that if a person is pregnant to add the parent relationship and then see they can choose unborn child.

I could see a worker potentially thinking someone isn't a parent yet and then never seeing the unborn child option.

Reply:
You make a good point - definitely a thing we will look out for feedback for when we do the initial release.